path.go

```diff
@@ -14,12 +14,8 @@
 
 package zetcd
 
-import (
-	"path"
-)
-
 func mkPath(zkPath string) string {
-	p := path.Clean(zkPath)
```

Comment:
this path is directly coming from client. we should clean it up before use it. maybe we shall add the trailing slash back if exists instead of removing the clean step entirely.